src/App.tsx

```diff
@@ -1,11 +1,17 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
-import { Bytes, KeyType, PrivateKey } from '@greymass/eosio';
 import { App as APP } from 'tonomy-id-sdk';
+import { mobileCheck } from './utills/IsMobile';
 function App() {
+    const [jwt, setJwt] = useState('');
     useEffect(() => {
-        APP.onPressLogin(window).then((result) => console.log(result));
+        APP.onPressLogin(window).then((result) => {
+            if (result && mobileCheck()) {
+                setJwt(result);
+                // window.location.replace('tonomy-id://test/home?jwt=' + result);
```

Comment:
what does this comment imply?

should it be removed? or change to //TODO?

Reply:
this should be added when we stop testing if comment removed u will automatically be redirected